electron/main.ts

```diff
@@ -60,12 +60,15 @@ ipcMain.on("add-temp-file", (event, file: string) => {
 });
 
 ipcMain.on("get-app-data", event => {
+    const filePath = process.argv[process.argv.length - 1];
```

Comment:
If `process.argv.length <= 1`, is this something else than `filePath`?

If that is the case, maybe we should change to `let filePath` here, and move the logic on line 71 to just after line 63? (In order for `filePath` be `null` in case it is not provided).